content/en/metrics/faq/updating-to-distribution-metrics-faq.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Updating to Distribution Metrics FAQ
+kind: faq
+is_beta: true
+further_reading:
+- link: "/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers/"
+  tag: "FAQ"
+  text: "How to Send Logs to Datadog via External Log Shippers?"
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- Unlocks new query functionalities:
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month) of their choosing.
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- Easier tag configuration and percentile aggregation workflows:
+  - The Metrics Summaray page now centrally manages all tag configuration and percentile aggregations.
+  - Use the the “Manage Tags” button on the Metrics Summary details side panel to manage all tag configurations for your distribution metrics.
+  - To add percentile aggregations to your distribution, navigate to the new percentiles section on the side panel, click *Edit* and select “Include percentile aggregations” for the same queryable tags defined as your baseline distribution. You’ll no longer need to specify an additional configuration rule.
+- More Intuitive number of Custom Metrics from Distributions with Percentiles:
+  The number of custom metrics emitted from additional percentile aggregations is no longer 5 percentile timeseries for every potentially queryable tag value combination. The new number of custom metrics emitted from distributions with percentiles is now counted more similarly to our other metric types:
+
+  | Metric Type                               | Query Type                                                                                                     | # of billable custom metrics emitted               |
```

Comment:
| Metric Type                               | Query Type                                                                                                     | # of custom metrics emitted               |